### ISSUE TITLE
Feature: Ability to serve existing files/directories long term

### ISSUE THREAD

**Issue by vRobM** [Seb3thehacker/sphinx-server#5]
When something is already in the 'filespath', it would be nice to be able to share it.

Same for directories (especially useful with webtorrent)

**Comment by Seb3thehacker** [comment]
Would you elaborate? I'm not sure I understand. Are you requesting a share button?

**Comment by vRobM** [comment]
The default flow is to first upload something to be able to share it..

However the server may already have files/directories to share in bulk.

**Comment by Seb3thehacker** [comment]
Yeah this would be really nice. This is going to be a low priority for now.
